src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -866,11 +866,6 @@ namespace winrt::TerminalApp::implementation
         //      UI yet. We'll display the error in _OnLoaded.
         _settingsLoadedResult = _TryLoadSettings();
 
-        if (FAILED(_settingsLoadedResult))
-        {
-            _settings = CascadiaSettings::LoadDefaults();
-        }

```

Comment:
Obviously, we can't call `LoadDefaults()` anymore so it makes sense to remove this branch. That said, can't we still hit these two `catch` branches in `_TryLoadSettings()` where we would've `FAILED(HR)`?

https://github.com/microsoft/terminal/blob/9a59b094680fe804e49d5bc995e011984c610e74/src/cascadia/TerminalApp/AppLogic.cpp#L827-L837

I see that the first branch also sets `_settingsLoadExceptionText` and I'm also concerned that now we're losing that.